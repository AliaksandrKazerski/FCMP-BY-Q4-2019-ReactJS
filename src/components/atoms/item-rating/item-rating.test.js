import React from 'react';
import { shallow } from 'enzyme';

import ItemRating from './index';

describe('ItemRating', () => {
  it("ItemRating should render correctly", () => {
  const myComponent = shallow(<ItemRating/>);
  
  expect(myComponent).toMatchSnapshot()
  });
});