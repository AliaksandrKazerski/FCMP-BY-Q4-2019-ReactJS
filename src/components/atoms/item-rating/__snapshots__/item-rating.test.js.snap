// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemRating ItemRating should render correctly 1`] = `ShallowWrapper {}`;
